op-node/node/node.go

```diff
@@ -40,6 +40,8 @@ type OpNode struct {
 	tracer    Tracer                // tracer to get events for testing/debugging
 	runCfg    *RuntimeConfig        // runtime configurables
 
+	coordinatorClient *client.CoordinatorClient // Coordinator RPC
```

Comment:
better to only put `CoordinatorClient` in `struct Sequencer`, as coordinator only effects the behavior of sequencer(it belongs to sequencer level, not node level), so we can limit the related code into `sequencer.go`

Reply:
Updated at https://github.com/bnb-chain/opbnb/pull/13/commits/17113eba408dd7f7208f9eac785bfc2871de6a1e. PTAL.